sound/soc/sof/imx/common.c

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// Copyright 2020 NXP
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
```

Comment:
@IulianOlaru249 as we discussed no need for these two lines, SPDX identifier at the top is enough.